test/JDBC/expected/TestSpatialFun-vu-prepare.out

```diff
@@ -0,0 +1,219 @@
+CREATE TABLE YourTableTemp ( ID INT PRIMARY KEY, PointColumn geometry ); 
```

Comment:
1. Let's use a complete name: Test-spatial-functions-vu-*
2. Let's use consistent naming for all the objects which is to prefix each object name with the filename, something like TestSpatialFunction_geogtable1 etc.

Reply:
resolved